eZ/Bundle/EzPublishCoreBundle/Command/DebugConfigResolverCommand.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\VarDumper\Cloner\VarCloner;
+use Symfony\Component\VarDumper\Dumper\CliDumper;
+
+class DebugConfigResolverCommand extends ContainerAwareCommand
```

Comment:
@alongosz Actually, that would be bad practice: https://github.com/ezsystems/ezpublish-kernel/pull/2650#issuecomment-495914309

Reply:
But the actual issue is with the injected services, not the command itself? This creates a lot of technical debt and thus adds extra work for @webhdx on top of what he has already done...

Reply:
> But the actual issue is with the injected services, not the command itself?

So in this case it would be fine, config resolver is anyway a lazy loaded register, and siteacess is updated on change inline.

But my point was in regards to for instance injecing contentService. See the link and comment on that there maybe.

ng> This creates a lot of technical debt and thus adds extra work for @webhdx on top of what he has already done...

Not following

Reply:
Ok, thanks for clarifying.
In summery we will mark commands loading things like contentService and similar as lazy, and in this case we can safely inject resolver and SA.